### PR TITLE
Update pandoc to 2.8.0.1

### LINKED ISSUE #9: Pandoc version as parameter
Bravo(!) for designing these rules to supply pandoc; many Bazel rules out there make it the user's problem to have the underlying binaries in place on all build machines, creating opportunity for version skew and other ills.

But... the downside is that, for example here in this rule, it can be tough to keep up with versions of the underlying tool. It appears that `bazel-pandoc` is currently hardcoded to retrieve and use only Pandoc v2.3.1. Pandoc 2.8 just shipped, and there have been various other versions since 2.3.1.

**Therefore, a feature request**:

* Make `pandoc()` accept a version parameter
* Update `pandoc()` to know about recent versions

A downside of this approach: each new version of Pandoc requires a new `bazel-pandoc`. It is possible to work around that by allowing the user to supply a replacement mapping (from platform to URL+hash) so that a newer version of Pandoc can be used immediately with minor inconvenience. Such a mechanism can also be used by folks who have a requirement to fetch all tools from a blessed internal repository.

### PR DESCRIPTION
**Pull request by regisd**
Fixes #9

### CODE REVIEW

File: repositories.bzl [hunk 1]
```diff
@@ -1,60 +1,54 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-def pandoc_repositories():
-    http_archive(
-        name = "pandoc_bin_linux",
-        build_file_content = """
+PANDOC_VERSION = "2.8.0.1"
```
Comment (patbro) [inline repositories.bzl:3]: It's great to have the Pandoc version as a define, but you still have to change the `sha256` when updating to a never version of Pandoc as well. 

The `sha256` may be changed to a define as well, but currently I don't see any issue in leaving it as is.

**Review by patbro — APPROVED** [review]

**Comment by regisd** [comment]
I thought I changed all the fingerprints. I even stupidly had to download
them all for that, I wish github were showing it with the releases. Maybe I
forgot to push, I'll check.

Patrick van Broeckhuijsen <notifications@github.com> schrieb am Do., 5.
Dez. 2019, 08:00:

**Comment by patbro** [comment]
I totally agree with you... Usually I just `curl` the file and run `sha256sum`. It's a bit cumbersome, but it gets the job done.

File: repositories.bzl (hunk 1 above)
Comment (regisd) [reply repositories.bzl:3]: I verified an already did. Sorry that the diff doesn't make this clear.

**Comment by patbro** [comment]
This PR can be merged now since all needed changes are include/completed, right?

**Comment by regisd** [comment]
> This PR can be merged now since all needed changes are include/completed, right?

I think it works, yes.

**Pull request merged**